syntax = "proto3";

message SVMTrainerMessage {
  oneof value {
    SetTrainResult setTrainResult = 1;
    SetParamGrid setParamGrid = 2;
  }
}

message SetTrainResult {
  int32 version = 1;
  string params = 2;
  double score = 3;
  bytes model = 4;
}

message SetParamGrid {
  string grid = 1;
}
